packages/dapp-high-roller/src/data/types.ts

```diff
@@ -335,4 +335,19 @@ export namespace cf {
     appInstances: { [appInstanceId: string]: AppInstance };
     nodeProvider: NodeProvider;
   };
+  export type NodeProvider = {
```

Comment:
Can this package not have `nodeProvider` as a dependency so that type matching doesn't have to be exported like this?

Reply:
This file shouldn't exist. We couldn't get `@counterfactual/types` imported into the Stencil apps. Maybe now it is fixed and we should try again